ionet-setup.sh

```diff
@@ -32,6 +32,16 @@ else
 # Check if nvidia-smi is available and working
     if command -v nvidia-smi && nvidia-smi | grep CUDA | grep -vi 'n/a' &>/dev/null; then
         echo "CUDA drivers already installed as nvidia-smi works."
+
+        # Extract the CUDA version from nvidia-smi output and check if it meets the minimum requirement
+        cuda_version=$(nvidia-smi | grep "CUDA Version" | sed 's/.*CUDA Version: \([0-9]*\.[0-9]*\).*/\1/')
+        if [ "$(echo "$cuda_version >= 11.8" | bc -l)" -eq 1 ]; then
```

Comment:
If I'm correct, that represents both versions as decimal fraction numbers... it won't compare correctly in all cases, e.g. this comparison would fail for `10.80 > 10.8`.

I suggest using `sort`, like https://stackoverflow.com/a/4046127